src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
 // write code here
+function sortList(list) {
+  const children = Array.from(list.children);
+
+  children.sort(sortFunc);
```

Comment:
The `sortFunc` is used here to sort the children. Ensure that all list items have the `data-salary` attribute, as missing attributes could lead to unexpected results.